docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/heroku.md

```diff
@@ -11,8 +11,144 @@ This is a step-by-step guide for deploying a Strapi project on [Heroku](https://
 ### Heroku Install Requirements
 
 - You must have [Git installed and set-up locally](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup).
+- You must have a [Github account](https://github.com/signup) if you are deploying with the web interface.
 - You must have a [free Heroku account](https://signup.heroku.com/) before doing these steps.
 
+## Deploying with the web interface
```

Comment:
This shouldn't be the default, it should be moved below the CLI.

We recommend the CLI over the GUI

Reply:
Ok. I have made some adjustment.